apps/sr-frontend/src/app/Header.tsx

```diff
@@ -42,6 +51,20 @@ const lightCones: ITab = {
   textSuffix: <LightConeChip key="lightConeAdd" />,
 }
 
+function RelicChip() {
+  const { database } = useDatabaseContext()
+  const [dirty, setDirty] = useForceUpdate()
+  useEffect(
+    () => database.relics.followAny(() => setDirty()),
```

Comment:
This is more of a nit to how we do this in GO, I think these gets rerendered every time we change an artifact/relic, but it should rerender every time an artifact/relic is added/removed (since it only display the total). 

It doesn't have to be this PR, but I think it make sense to use the `DataManagerCallback.reason` string, and check against `'remove' | 'new'`. Would also make sense to store `total` as a state instead of `useForceUpdate`, so when total is set as the same value again, it doesnt trigger a revalue.